ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java

```diff
@@ -20,11 +22,13 @@
     private static final String EXTRA_ORIENTATION = "adOrientation";
     static final String EXTRA_ALL_PLACEMENTS = "allPlacements";
     static final String EXTRA_PLAY_PLACEMENT = "playPlacement";
+    static final String UUID_KEY = "uniqueVungleRequestKey";
 
     private final Bundle mBundle = new Bundle();
 
     public VungleExtrasBuilder(@Nullable @Size(min = 1L) String[] placements) {
         mBundle.putStringArray(EXTRA_ALL_PLACEMENTS, placements);
+        mBundle.putString(UUID_KEY, UUID.randomUUID().toString());
```

Comment:
@HuFangshuai This was done by me , just saw this comment posted on upstream PR https://github.com/googleads/googleads-mobile-android-mediation/pull/212#discussion_r385912324
which asks that we should also provide an explicit key so that Publishers can also have an option to add the unique 
 request key.
We should add another public method to setBannerRequestUUID , and during build check if mBundle already has the key, and if not add this code in there.
This will provide a way for Publishers to add the key, and by default in absence of the key we would still have logic to generate one for every new instance

Reply:
fixed.